sjb/config/test_cases/test_branch_jenkins_images.yml

```diff
@@ -16,5 +16,8 @@ extensions:
       repository: "jenkins"
       timeout: 7200
       script: |-
+        # this docker tag is for our 3.6 branch; we did not introduce
+        # the use of the ose image until 3.7
+        docker tag registry.access.redhat.com/openshift3/ose:v3.6 registry.access.redhat.com/rhel7
```

Comment:
i'd be more inclined to do this in the same place where the fake ose:v3.6 image is being created, so someone doesn't go change/remove that logic and break this.

Reply:
||    how about a comment in that file, but the actual tagging in this file, so as the minimize where ||the tagging occurs ?

| i'm not sure why minimizing where tagging occurs is a goal. I would set the goal as minimizing |where "constructing the subscribed base images/tags" occurs. However you're the one most likely |to have to maintain this, so it's up to you.


I'm worried about a new job getting defined that wants to use `registry.access.redhat.com/rhel7` as its from image and the rhel base job as its parent, and getting tripped up by our tagging.  Is that not a valid concern?

@stevekuznetsov - any opinions ?

Reply:
that new job shouldn't extend the "rhel_base_images" job then. The only purpose of that job is to build images that have subscriptions+necessary repos built in.

(admittedly the job name/description could be better)